### PR TITLE
[#436] Removed aria-label from link with 'is_new_window'.

### PR DESCRIPTION
**Pull request by alan-cole**
https://github.com/civictheme/uikit/issues/436

## Checklist before requesting a review

- [x] I have formatted the subject to include the issue number
  as `[#123] Verb in past tense with a period at the end.`
- [x] I have provided information in the `Changed` section about WHY something was
  done if this was a bespoke implementation.
- [x] I have performed a self-review of my code.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] I have run new and existing relevant tests locally with my changes,
  and they have passed.
- [x] I have provided screenshots, where applicable.

## Changed

1. Removed aria-label from link with 'is_new_window'. See issue for reasoning.

## Screenshots
Screenshot 2024-12-01 at 11 03 18 am

### CODE REVIEW

**Comment by richardgaunt** [comment]
@alan-cole I am going to add to next release as I want this tested closely etc.

**Review by richardgaunt — APPROVED** [review]

File: components/01-atoms/link/link.twig [hunk 1]
```diff
@@ -50,7 +50,7 @@
     class="ct-link {{ modifier_class -}}"
     {% if url is not empty %}href="{{ url }}"{% endif %}
     {% if title is not empty %}title="{{ title }}"{% endif %}
-    {% if is_new_window %}target="_blank" aria-label="Opens in a new tab"{% endif %}
+    {% if is_new_window %}target="_blank"{% endif %}
```
Comment (joshua-salsadigital) [inline components/01-atoms/link/link.twig:53]: @alan-cole  Can you review this issue if its related.  https://www.drupal.org/project/civictheme/issues/3488286
The MR has code updated around the same lines. 

https://git.drupalcode.org/project/civictheme/-/merge_requests/14/diffs#d877c5d98f677ff76fdcbb87250a8f99f0c2cac7_54_53

Comment (alan-cole) [reply components/01-atoms/link/link.twig:53]: Yes, this fix should fix that issue.

**Review by richardgaunt — APPROVED** [review]
@alan-cole Thanks Alan this looks good to me.

**Pull request merged**